kata02/chop_test.go

```diff
@@ -27,12 +28,22 @@ func TestChop1(t *testing.T) {
 		{7, medHaystack, 3},
 		{0, medHaystack, -1},
 		{8, medHaystack, -1},
+		{7, largeHaystack, 3},
+		{19, largeHaystack, 9},
+		{35, largeHaystack, 18},
+		{99, largeHaystack, -1},
+		{101, largeHaystack, -1},
 	}
 
-	for _, table := range tables {
-		result := Chop1(table.needle, table.haystack)
-		if result != table.result {
-			t.Errorf("Chop1 of (%v) in (%v) was incorrect, got: %v, want: %v.", table.needle, table.haystack, result, table.result)
+	chops := []Chop{Chop1, Chop2}
+
+	for i, chop := range chops {
```

Comment:
changed my test from the original commit to iterate over the chop functions currently available.